docs/source/class_basics.rst

```diff
@@ -111,7 +111,7 @@ annotation, it is considered an untyped method:
 Class attribute annotations
 ***************************
 
-You can use a ``ClassVar[t]`` annotation to explicitly declare that a
```

Comment:
I'm unsure whether a reference is allowed or necessary here. Also, shouldn't the type be a capital `T`?

Reply:
I think it's useful. The `t` is intentionally lowercase, since it's not a type variable -- in this context (and occasionally elsewhere) it's just any type.